spec/features/post_peep_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+require_relative 'helpers/session'
+
+feature "User adds a new peep" do
+
+  include SessionHelpers
+
+   before(:each) do
+    User.create(:name => 'tester',
+                :username => 'username_test',
+                :email => "test@test.com",
+                :password => 'test',
+                :password_confirmation => 'test')
+    end
+
+  scenario "when logged in and browsing the homepage" do
+    sign_in('test@test.com', 'test')
+    expect(Peep.count).to eq(0)
+    visit '/'
+    post_peep("This is my second peep")
+    expect(Peep.count).to eq(1)
```

Comment:
this will fail because you visit '/' before trying to post a peep - your ``` get '/' ``` controller only renders index.erb, not peeps/new.erb